.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: python
-sudo: false
+sudo: required
+dist: xenial
 python:
   - "2.7"
   - "3.4"
   - "3.5"
   - "3.6"
+  - "3.7"
```

Comment:
This still refers to *3.7.1*. You can replace it with either `3.7.3` (released [2019-03-25](https://pythoninsider.blogspot.com/2019/03/python-373-is-now-available.html)) or `3.7-dev` (*3.7.3+* at the moment).

Reply:
Applied. Thanks!